docs/UserGuide.md

```diff
@@ -346,6 +346,27 @@ or search the specific patient using [`find t/patient` command](#finding-entries
 
 ![tut-view-patient-2](images/patientug/tut_view_patient_2.png)
 
+### Clear all patient entries: `clear t/patient`
+
+Clears all patient entries from GoMedic.
+
+Format: `clear t/patient`
+
+**Example:**
+
+&#8291;1. Type the command `clear t/patient` into the command box.
+
+![tut-view-patient-1](images/patientug/tut_clear_patient_1.png)
+
+&#8291;2. Press `Enter` and all the patients will be deleted.
+
+![tut-view-patient-2](images/patientug/tut_clear_patient_2.png)
+
+**Note:**
+
+This will also delete all corresponding appointments since there are no more patients in GoMedic.
```

Comment:
You can put this using the precaution or alert box using the <div class="alert alert-info"> to highlight it